app/statistics/server/lib/statistics.ts

```diff
@@ -52,11 +56,142 @@ const getUserLanguages = async (totalUsers) => {
 
 const { db } = MongoInternals.defaultRemoteCollectionDriver().mongo;
 
+type STATISTICS = {
```

Comment:
Instead of adding a new type, update IStats type with new keys and then you'll be able to insert it on L602

Reply:
Damn, didn't see IStats.